Emby.Dlna/Didl/DidlBuilder.cs

```diff
@@ -113,6 +113,8 @@ public string GetItemDidl(BaseItem item, User user, BaseItem context, string dev
 
                     writer.WriteFullEndElement();
                     // writer.WriteEndDocument();
+
+                    writer.Flush();
```

Comment:
You sure this is necessary? When XmlWriter is disposed it should call Close on itself and Flush.

Reply:
Backported from the plugin where the using is no longer in brackets (as suggested by compiler).

Have changed to a warning comment - as following the analyser's suggestion breaks it.